dotCMS/src/main/java/com/dotcms/concurrent/DotConcurrentFactory.java

```diff
@@ -342,6 +379,156 @@ public IdentifierStripedLock getIdentifierStripedLock(){
        return this.identifierStripedLock;
     }
 
+    /**
+     * {@link SubmitterConfigCreator} builder
+     */
+    public static class SubmitterConfigCreatorBuilder {
+
+        private ThreadFactory            threadFactory;
+        private RejectedExecutionHandler rejectedExecutionHandler;
+        private Boolean allowCoreThreadTimeOut;
+        private Integer poolSize;
+        private Integer maxPoolSize;
+        private Long    keepAliveMillis;
```

Comment:
Codacy found an issue: [Field keepAliveMillis has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6265363)